internal/pkg/build/stage.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
```

Comment:
Update copyright:

```suggestion
// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Or:

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```